api_examples/sequential_learning_api_tutorial.ipynb

```diff
@@ -0,0 +1,692 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Standard packages\n",
+    "from os import environ\n",
+    "from time import sleep\n",
+    "\n",
+    "# Third-party packages\n",
+    "import numpy as np\n",
+    "import matplotlib.pyplot as plt\n",
+    "import seaborn as sns\n",
+    "\n",
+    "from pypif import pif\n",
+    "from pypif.obj import *\n",
+    "from citrination_client import CitrinationClient\n",
+    "from citrination_client.models.design import Target\n",
+    "\n",
+    "sns.set_context(\"talk\")\n",
+    "sns.set_style(\"whitegrid\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setting up a client\n",
+    "\n",
+    "Make sure to properly set your **environment variable** to hold your API key."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = CitrinationClient(\n",
+    "    environ.get(\"CITRINATION_API_KEY_PUBLIC\"),\n",
+    "    \"https://citrination.com\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Data generation\n",
+    "\n",
+    "Since we aren't using a real laboratory, we need access to a quick way to generate \"correct\" measurements. A simple placeholder here is to use a function that sums the squares of its inputs. The goal, in this case, will be to find the global minimum. \n",
+    "\n",
+    "In a real example, we could minimize or maximize any output: compressive strengths, conductivities, and so on."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def toy_func(inputs):\n",
+    "    func_sum = 0.0\n",
+    "    for inp in inputs:\n",
+    "        func_sum += inp**2\n",
+    "    return func_sum\n",
+    "\n",
+    "toy_x = [np.random.normal(loc=3.0, scale=1.0, size=(1, 2))[0] for x in range(10)]\n",
+    "toy_y = [toy_func(x) for x in toy_x]\n",
+    "\n",
+    "initial_best_measured_value = min(toy_y)\n",
+    "\n",
+    "print(f\"Best (lowest) value in initial training set: {initial_best_measured_value}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can plot the initial training set, and color it by the function value"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Plot initial training set, colored by toy function values\n",
+    "\n",
+    "plt.scatter(np.array(toy_x)[:,0], np.array(toy_x)[:,1],\n",
```

Comment:
This is so pretty! <3 Can we add a colorbar? Optional, but for a second I wasn't sure which color was optimal (and not realizing that we're looking for points close to origin)